gammapy/estimators/excess_map.py

```diff
@@ -33,18 +33,18 @@ def convolved_map_dataset_counts_statistics(dataset, kernel, mask):
         background = dataset.background * mask
         background.data[dataset.acceptance_off.data == 0] = 0.0
         n_off = dataset.counts_off * mask
-
-        background_conv = background.convolve(kernel.array)
-        n_off_conv = n_off.convolve(kernel.array)
-
         npred_sig = dataset.npred_signal() * mask
-        mu_sig = npred_sig.convolve(kernel.array)
+
+        background_convolve = background.convolve(kernel.array)
```

Comment:
I think you would convolve the background (alpha*counts_off) only if correlate_off is set to True no?

Reply:
If I'm right in this case the background is used as an estimation of the acceptance for On events, as we correlate On events, we should also correlate the acceptance.

Reply:
The ON acceptance is given by the `dataset.acceptance` member. 
`dataset.background = dataset.alpha * dataset.counts_off` in the case of a `MapDatasetOnOff`.